docker-compose.yml

```diff
@@ -33,6 +33,7 @@ services:
       - "3000:3000"
     environment:
       - REDIS_HOST=redis
+        SESSION_SECRET=session-secret
```

Comment:
shouldnt there be a "-" ?

Reply:
thanks @steverendell, I add this in and create a PR.